gradle/rewrite.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: 'org.openrewrite.rewrite'
+
+rewrite {
+	activeRecipe(
+			"tech.picnic.errorprone.refasterrules.StringRulesRecipes",
```

Comment:
where does this come from?

Reply:
Defined [here](https://docs.openrewrite.org/recipes/tech/picnic/errorprone/refasterrules/stringrulesrecipes), with delegation to [these rules](https://error-prone.picnic.tech/refasterrules/StringRules/) (source code [here](https://github.com/PicnicSupermarket/error-prone-support/blob/4bac3f56c9a48a392861cce9591fa4f352aadfca/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StringRules.java)).

Reply:
yes thanks @Stephan202, with the help from the friends of rewrite, picnic, and google we able to prevent around 600 rules covered in the refaster rules.

Just need to activate them.

Im wondering how to potentially activate the entire set, as errorprone is errorprone? Its clear that this amount will take tremendous computing effort, making it an dedicated cloud only job.

Reply:
Kidly consider:

- https://github.com/diffplug/spotless/pull/2576

```
running active recipes: tech.picnic.errorprone.refasterrules.NullRulesRecipes
<============-> 99% EXECUTING [1h 38s]
```

The string recipe is a quite bigger composite recipe so its quite a challenge to apply once, after the initial migration it should decrease a litte bit.

Reply:
(Going on holiday, so can't further support for now.)

For advice around OpenRewrite performance, I'll defer to @timtebeek. One alternative you could try is our Refaster-to-Error Prone bridge (`refaster-runner`) as described [here](https://github.com/PicnicSupermarket/error-prone-support/#gradle). To start light you'll then likely have to disable all our other checks and run with `-XepOpt:Refaster:NamePattern=^StringRules\$.*`. Note that this will perform only a single pass (unlike OpenRewrite, IIUC), but after the repo is cleaned up, that's generally not an issue.